env.config.js

```diff
@@ -13,14 +13,15 @@ const production = {
 }
 
 const validate = (config) =>
-  ['API_URL'].forEach((param) => {
+  ['API_URL', 'API_KEY'].forEach((param) => {
     if (config[param] == null) throw new Error(`${param} is not configured.`)
   })
 
 const env = { local, development, production }
 const config = {
   ...env.production,
   ...env[NODE_ENV],
+  API_KEY: process.env.CORE_API_KEY,
```

Comment:
@mcancellieri I am not sure about this mapping. I prefer API_KEY in the code for consistency. Also, I assume, we use only one API, so there is no need to clarify it with prefix but who knows...

I tend to choose either one or another.

Reply:
ok, my vote goes for API_KEY too...
The CORE part is redundant, if you will need other API Keys you can prefix them.

Reply:
Thank you for your comment. I will change them on Monday 🙂